### ISSUE TITLE
add url "schema" to organism in database?

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/py3-adage-backend#39]
Right now, I'm hard-coding in how genes link to pseudomonas.com based on the url param schema that site expects.

Casey suggested having the url be "abstracted out" from the frontend, and put into the database itself, basically having each organism define a useful url based on systematic id or some other gene property.

Not exactly sure the technical details on how we'd design this. I also don't see it as a high priority given that so many things are already hard coded assumptions. But it's something to think about in the future.

Currently, the gene links are like this: `pseudomonas.com/feature/show/?locus_tag=[SYSTEMATIC NAME]` Maybe each organism could have a template just like that, like: `organism: { id: 1, urlTemplate: 'somesite.com/lookup/?name=%systematicName%' }`

**Comment by dongbohu** [comment]
It is doable. To make the backend more consistent, the field name in the URL schema probably should be the exact field name in the model.

**Comment by dongbohu** [comment]
When this schema field is in the database, the web API can compile the exact URL and provide it to the frontend.

**Comment by dongbohu** [comment]
`adage-frontend/src/pages/gene/link.js`:
```
'http://www.pseudomonas.com/feature/show/?locus_tag=' + gene.systematicName
```

Management command to add/update the organism:
```sh
./manage.py organisms_create_or_update \
--taxonomy_id=208964 \
--common_name="Pseudomonas aeruginosa" \
--scientific_name="Pseudomonas aeruginosa" \
--url_template="http://www.pseudomonas.com/feature/show/?locus_tag=<systematic_name>"
```

**Comment by dongbohu** [comment]
Fixed by PR #44.
